.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+# Contributing to TDS Community
+
+Thank you for your interest in contributing! There are a few ways you can help:
+
+* Comment on [existing issues](https://github.com/telusdigital/tds-community/issues) or [open a new one](https://github.com/telusdigital/tds-community/issues/new)
+* Propose changes to documentation. (Did you know, you can propose changes [without leaving GitHub](https://help.github.com/articles/editing-files-in-your-repository/)?)
+* Add features or patches to [existing components](https://github.com/telusdigital/tds-community/tree/master/packages)
+* Add new components! _Read on_
+
+## Table of contents
+
+* [Community component criteria](#community-component-criteria)
+* [What makes a good community component?](#what-makes-a-good-community-component)
+* [Process](#process)
+* [Designer guide](#designer-guide)
+* [Developer guide](#developer-guide)
+  * [Codebase overview](#codebase-overview)
+  * [Environment setup](#environment-setup)
+  * [Making commits and versioning packages](#making-commits-and-versioning-packages)
+* [Making pull requests](#making-pull-requests)
+
+## Community component criteria
+
+Before designing or contributing new components, confirm the following criteria in order to qualify
+a viable community component.
+
+A community component:
+
+1.  **Must** have an identified use case in at least 2 unique applications
+    * Community components have the most impact when shared across multiple situations in many applications.
+2.  **Must** be brand aligned and assessed by Design Direction (if applicable)
+    * New user experience patterns must involve Design Direction to preserve a high-quality and consistent end customer experience.
+3.  **Must** not include business logic or proprietary information
+    * The presence of these things limits the breadth of reuse for a component.
+    * Design system components are focused on reusable user experience patterns. Keep business logic, API calls, content, or other application-specific behaviour in the application.
+    * Any user-facing copy, such as English phrases, must have the ability to be overridden in order to support multilingual applications or changes in content direction
```

Comment:
Recommend taking this phrase out. This is called out in the "Implementation Specifics" section below: "DON'T hardcode content without giving consumers the ability to override it. What about French?"

My intention was to keep the "Criteria" as high level and short as possible for brevity and clarity. And then give more details below.

Given that, I recommend removing this point here.